### PR TITLE
Add content for Gradle Build Server

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

**Review by nickzhums — APPROVED** [review]

File: docs/java/java-build.md [hunk 1]
```diff
@@ -128,7 +128,24 @@ There are several ways to create a Maven project:
...
+- **Visual Interface:** Allows you to view and manage Gradle tasks and project dependencies, and run Gradle tasks directly within VS Code.
+- **Gradle Language Server:** Provides an improved authoring experience for Gradle build files, including syntax highlighting, error reporting, and auto-completion.
+
+### Gradle Build Server
+By default, if you have installed the Gradle for Java extension, the Gradle Build Server is used to import Gradle projects. You can toggle the Gradle Build Server on or off by setting `java.gradle.buildServer.enabled`.
+
+You can view the Gradle build output in the Build Server for Gradle (Build) output channel and trace the interactions between VS Code and the Gradle Build Server in the Build Server for Gradle (Log) output channel.
+
+### Delegate tests to Gradle
+The extension supports delegate the test execution to Gradle. You can configure the test profile you want to use in the Testing explorer.
```
Comment (ntrogh) [inline docs/java/java-build.md:142]:
```suggestion
The extension supports delegating the test execution to Gradle. You can configure the test profile you want to use in the Testing explorer.
```

File: docs/java/java-build.md [hunk 2]
```diff
@@ -128,7 +128,24 @@ There are several ways to create a Maven project:
...
+
+<video autoplay loop muted playsinline controls title="Delegate tests to Gradle">
+  <source src="/docs/java/java-build/delegate-test-to-gradle.mp4" type="video/mp4">
+</video>
+
+> Note: Makes sure you have installed the [Test Runner for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) extension to use this feature.
```
Comment (ntrogh) [inline docs/java/java-build.md:148]:
```suggestion
> Note: Make sure you have installed the [Test Runner for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) extension to use this feature.
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
A couple of text edits. Otherwise, LGTM.

**Review by ntrogh — APPROVED** [review]
Looks good and can be merged.

**Review by lramos15 — APPROVED** [review]

**Comment by ntrogh** [comment]
@jdneo Let me know if you're ready to have these changes merged.

**Comment by jdneo** [comment]
Thank you @ntrogh, the change can be merged.

**Pull request merged**